core/src/main/java/hudson/Main.java

```diff
@@ -83,7 +86,8 @@ private static String getHudsonHome() {
     }
 
     /**
-     * Run command and place the result to a remote Hudson installation
+     * Run command and send result to {@code ExternalJob} in the {@code external-monitor-job} plugin.
+     * Obsoleted by {@code SetExternalBuildResultCommand} but kept here for compatibility.
      */
     public static int remotePost(String[] args) throws Exception {
```

Comment:
Mark it as deprecated and NoExternalUse? https://github.com/jenkinsci/external-monitor-job-plugin/blob/c233e945f85d842406ce35e31c7a070f1340e222/src/main/java/hudson/cli/SetExternalBuildResultCommand.java does not call it